projects/1.20/assets/modrinth-deepdrilling/deepdrilling/lang/zh_cn.json

```
@@ -0,0 +1,63 @@
+{
+  "block.deepdrilling.andesite_drill_head": "安山岩制钻头",
+  "block.deepdrilling.asurine_node": "皓蓝石矿脉",
+  "block.deepdrilling.asurine_node.tooltip.behaviour1": "_皓蓝石，凝灰岩，方解石_",
+  "block.deepdrilling.asurine_node.tooltip.condition1": "伴生产物",
+  "block.deepdrilling.asurine_node.tooltip.summary": "在_寒冷生物群系_的_基岩层_中生成",
+  "block.deepdrilling.blank_module": "空白模块",
+  "block.deepdrilling.blank_module.tooltip.behaviour1": "_没什么用_... 但可以用来_连接_其他模块。",
+  "block.deepdrilling.blank_module.tooltip.condition1": "连接至钻芯时",
+  "block.deepdrilling.blank_module.tooltip.summary": "一种_钻探模块_，可与_钻芯_连接。",
+  "block.deepdrilling.brass_drill_head": "黄铜制钻头",
+  "block.deepdrilling.collection_filter": "收集模块",
+  "block.deepdrilling.collection_filter.tooltip.behaviour1": "可以_收集采掘出的矿物_，但存储有上限，多余的矿物会从侧面的孔中_溢出_。",
+  "block.deepdrilling.collection_filter.tooltip.behaviour2": "可取出_采掘出的矿物_",
+  "block.deepdrilling.collection_filter.tooltip.condition1": "连接至钻芯时",
+  "block.deepdrilling.collection_filter.tooltip.condition2": "对此模块右键",
```

`右键`不是一个完整的状语，需要一个动作补全

```suggestion
  "block.deepdrilling.collection_filter.tooltip.condition2": "对此模块右击",
```